reco_utils/dataset/movielens.py

```diff
@@ -42,73 +70,174 @@ def path(self):
     def has_header(self):
         return self._has_header
 
+    """ Item (Movie) file """
+
+    @property
+    def item_separator(self):
+        return self._item_sep
+
+    @property
+    def item_path(self):
+        return self._item_path
+
+    @property
+    def item_has_header(self):
+        return self._item_has_header
+
 
+# 10m and 20m data do not have user data
 _data_format = {
```

Comment:
if what you want is a constant dict, `_data_format` should be `DATA_FORMAT`, same with `_genres`. Is that what you want?